lib/big_brother/cluster.rb

```diff
@@ -1,6 +1,15 @@
 module BigBrother
   class Cluster
-    attr_reader :fwmark, :scheduler, :check_interval, :nodes, :name, :ramp_up_time, :nagios
+    attr_reader :fwmark, :scheduler, :check_interval, :nodes, :name, :ramp_up_time, :nagios, :backend_mode
+    ACTIVE_PASSIVE_CLUSTER = "active_passive"
+
+    def self.create_cluster(name, attributes)
```

Comment:
Not positive this is best place to put a constructor method, because it makes the interface for this type confusing... An external client of this type can still call the initializer _or_ the `create_cluster` method, more so it creates a dependency on the sub-type (ActivePassiveCluster) from the parent type.